README.markdown

```diff
@@ -23,6 +23,11 @@ Note: We still have some problems with FreeBSD, because there is no pre-built bi
 Note: On OS X, if you do not have Xcode installed and you do not wish to download the ~4.3GB file, you can install the `Command Line Tools`. You can check out this blog post on how to just that:
  - [How to Install Command Line Tools in OS X Mavericks & Yosemite (Without Xcode)](http://osxdaily.com/2014/02/12/install-command-line-tools-mac-os-x/)
 
+Note: On OS X, if you have/had a "regular" nodejs installed and want to install modules globally, keep in mind that:
```

Comment:
let's change this phrase to `"system" node` - `nodejs` is ubuntu's legacy way to refer to the apt package, and "regular" has connotations i don't like :-) (same below)